sql/test_data_second_domain.mysql

```diff
@@ -963,4 +963,3 @@ INSERT INTO civicrm_navigation
 VALUES
     ( @domainID, CONCAT('civicrm/report/instance/', @instanceID,'&reset=1'), 'Mailing Detail Report', 'Mailing Detail Report', 'administer CiviMail', 'OR', @reportlastID, '1', NULL, @instanceID+2 );
 UPDATE civicrm_report_instance SET navigation_id = LAST_INSERT_ID() WHERE id = @instanceID;
-UPDATE civicrm_domain SET version = '4.6.alpha1';
```

Comment:
Unfortunately this file was full of trailing whitespace; the only real change is to remove this ugly hack which was causing all tests to run with a very very wrong domain version. Honestly I don't understand how our tests were passing at all with this problem (note that the purpose of this file is ostensibly to mock a second domain but this query clobbers the version of the main domain too. OOPS!)

Reply:
Ooof, that's fun. I suppose technically it didn't matter before. 🤷‍♂️ 

I see this patch changes the version from 4.2 to 4.6 I've posted an alternative #17734 which should avoid the need for periodic tweaks.